charts/openstack-cloud-controller-manager/templates/daemonset.yaml

```diff
@@ -2,9 +2,8 @@ apiVersion: apps/v1
 kind: DaemonSet
 metadata:
   name: {{ include "occm.name" . }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
   namespace: {{ .Release.Namespace }}
-  labels:
-    {{- include "occm.labels" . | nindent 4 }}
```

Comment:
Will this give use the same labels here?

Reply:
Yes, this will result in the same labels

Reply:
I'm looking at CI results and I only see this label added to the DaemonSet:
```
Labels:         k8s-app=openstack-cloud-controller-manager
```
See for yourself in the "Describe failed openstack-cloud-controller-manager" step: https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/cloud-provider-openstack/2519/openstack-cloud-controller-manager-e2e-test/1749846521752326144/build-log.txt

Reply:
I don't know how CI comes up with that, but I would say it's technically impossible for this label to exist, neither in the old version nor my new one 😅

Reply:
Okay, CI gets that from https://github.com/kubernetes/cloud-provider-openstack/blob/f7abe101ed041c8867d5fd1faa8351044f40a9cb/manifests/controller-manager/openstack-cloud-controller-manager-ds.yaml, so I guess we can disregard this.